Mage.Server.Plugins/Mage.Player.Human/src/mage/player/human/HumanPlayer.java

```diff
@@ -2243,7 +2238,7 @@ private boolean suppressAbilityPicker(ActivatedAbility ability, Game game) {
             }
 
             // hide on alternative cost activated
-            if (!getCastSourceIdWithAlternateMana().contains(ability.getSourceId())
+            if (!getCastSourceIdWithAlternateMana().getOrDefault(ability.getSourceId(), new HashSet<>()).contains(MageIdentifier.Default)
```

Comment:
Use `Collections.emptySet()` instead of creating a `new HashSet<>()` when all you're going to do is check `contains()`.